multipatch_analysis/synaptic_dynamics.py

```diff
@@ -520,6 +520,20 @@ def plot_model_results(self, model=None, fit=None):
         return rel_plots
 
     def cross_talk(self):
+        """#TODO: Luke you should edit this
```

Comment:
Return the average electrode crosstalk artifact amplitude.

Crosstalk amplitude is measured as the difference between means of 1 ms windows immediately before and after the onset of the presynaptic stimulus pulse, averaged across all pulses.

Reply:
There's a bug in this code; it should use the onset time of the pulse rather than assuming 10 ms.